src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -207,6 +208,9 @@ public void storeProfile(Profile profile) {
     Entity profileEntity = new Entity("Profile", profile.getEmail(), user);
     profileEntity.setProperty("email", profile.getEmail());
     profileEntity.setProperty("name", profile.getName());
+    if(profile.getProfilePicURL() != null) {
+      profileEntity.setProperty("profile_pic", profile.getProfilePicURL());
+    }
```

Comment:
fix formatting here. most modern IDEs let you do it in one go but you can just do it manually.